app/workers/daily_grouping_updater.rb

```diff
@@ -0,0 +1,15 @@
+class DailyGroupingUpdater
+  include Sidekiq::Worker
+
+  def initialize
+    @system_account = Watcher.retrieve_system_account
+    PaperTrail.whodunnit = @system_account
+  end
+
+  def perform(app_name:)
+    groupings = Grouping.app_name(app_name).where(state: :acknowledged).where("latest_wat_at <= ?", 15.days.ago)
```

Comment:
Could we move the AR-related bits to the model / a service, so that the worker has no clue about that logic.

Also, let's pass the time range (the 15 days) in the `perform` method signature. It's gonna be easy to reload stuff for a longer period if something happens, or just to backfill history. 👍